views/crawl-cache-page.php

```diff
@@ -4,6 +4,7 @@
     <thead>
         <tr>
             <th>URLs in Crawl Cache</th>
+	          <th>Page MD5 Hash</th>
```

Comment:
Not sure user will want to see the hashes vs the paths in the view. Move to 2nd column?

Reply:
I think this is the second column if that's what you meant. It comes in handy when the pages aren't getting loaded and all the hashes are the same (which they shouldn't be)